src/main/java/seedu/address/logic/parser/LinkCommandParser.java

```diff
@@ -73,22 +73,22 @@ private LinkCommand staticParse(String args) throws ParseException {
     }
     public String getStaticUsageInfo() {
         return "Link command usage: link "
-                + "-student [STUDENT_NAME]"
-                + "-lesson [LESSON_NAME]"
+                + "-student STUDENT_NAME"
+                + "-lesson LESSON_NAME"
```

Comment:
i think this part is fine, just maybe need to specify in UG that STUDENT_NAME and LESSON_NAME follow the same constraints as NAME parameter

Reply:
Emm, sure